blogposts/announcing-sourcegraph-3.14.md

```diff
@@ -0,0 +1,258 @@
+The Sourcegraph permissions syncing has gotten more sophisticated to better handle repository permissions at scale. Rather than updating a user’s permissions when they log in and potentially blocking them from seeing any search results, Sourcegraph now syncs these permissions opportunistically, and as part of a background service. Users are now able to see results from their search queries immediately. Currently, this improvement is supported for GitLab and Bitbucket Server, with GitHub and other code hosts coming soon. 
+
+Enable this feature by adding `"permissions.backgroundSync": {"enabled": true}` in your site configuration settings. 
+
+Keep your eyes out for improvements to the site admin pages to allow for better insight into when permissions were last updated for users, and the ability to force a refresh of permissions for a user or repository.
+
+## Forks and archived repositories are excluded from search results by default
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/398759697?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/398759697" target="_blank">View on Vimeo</a></p>
+</p>
+
+The default setting for including forks and archived repositories in your search results has changed from including by default to excluding them by default. This is because most often, users want to see the active and original repositories in their queries. You can still use `archived:yes` or `forks:yes` filters to show results that include archived and forked repositories.
+
+## Deploy and scale Sourcegraph with docker-compose
+
+Sourcegraph 3.14 introduces a new [docker-compose deployment model](https://docs.sourcegraph.com/admin/install/docker-compose). This is useful for organizations that want to scale Sourcegraph, but don’t use Kubernetes. For any customers running a single docker container and have experienced instability or issues with scaling, Docker-compose is likely the right solution for you!
+
+## Improved campaign creation, management, and syncing performance
+
+TODO: Video
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+The overarching experience of generating diffs, creating and reviewing a campaign, then managing the changes to closure has been improved.
+
+Using the `src-cli` to generate diffs and create a campaign plan has been improved:
```

Comment:
I would suggest moving the `src-cli` section below the "Behind the scenes,..." paragraph, then the "We've improved the experience" start continues nicely with the two paragraphs that explain the improvements in prose.